src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -369,4 +363,38 @@ public ApplianceBootstrapInformationElement getBootstrapinfo(BootStrapInfoProvid
             }
         };
     }
+
+    private DeviceMemberEntity getDeviceMembers(Long deviceId, Long vsMgrId, Long memberId, String memName) {
```

Comment:
again dont understand this method. device ID and vs mgr id are used in the same way. There should only be one id. This would simplify the method a by eliminating duplicate blocks

```
criteriaBuilder.equal(r.get("deviceId").get("id"), deviceId)

criteriaBuilder.equal(r.get("deviceId").get("id"), vsMgrId)
```

Reply:
Yes there is only one id and from the rest interface context, deviceId is passed and from OSC-->to plugin context mgrId is passed and both are the same. I will remove one of the parameters as you have proposed.